rainmaker_orchestrator/clients/kimi.py

```
@@ -0,0 +1,41 @@
+            response = self.client.chat.completions.create(
+                model=self.model,
+                messages=[
+                    {"role": "system", "content": system_prompt},
+                    {"role": "user", "content": prompt}
+                ],
+                temperature=0.3 if mode == "hotfix" else 0.7,
+                max_tokens=1000
+            )
+
+            return response.choices[0].message.content
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)